crates/rooch/src/commands/event.rs

```diff
@@ -29,10 +30,14 @@ pub enum EventSubCommand {
 
 #[derive(Debug, clap::Parser)]
 pub struct GetEventsByEventHandle {
+    /// Struct name as `<ADDRESS>::<MODULE_ID>::<STRUCT_NAME><TypeParam1?, TypeParam2?>`
```

Comment:
struct name doesn‘t contains `<TypeParam1?, TypeParam2?>`, just remove it as below
Struct name as `<ADDRESS>::<MODULE_ID>::<STRUCT_NAME>`

Reply:
StructTag can contain TypeParam. And `MODULE_ID = ADDRESS + MODULE_NAME`.

`ADDRESS::MODULE_NAME::STRUCT_NAME<TypeParam1?, TypeParam2?>`

 Remove the `<` to avoid the developer to confuse with TypeParam's '<'.